src/components/MUITypography/Typography.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import { Typography, TypographyVariant } from '@material-ui/core';
+import { boolean, select } from '@storybook/addon-knobs';
+import React from 'react';
+
+export default { title: 'Components/Typography' };
+
+const VariantOptions: TypographyVariant[] = [
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'h6',
+  'body1',
+  'body2',
+  'subtitle1',
+  'subtitle2',
+  'caption',
+  'button',
+  'overline',
+];
+
+const VariantValues: string[] = [
+  'Heading.',
+  'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur.',
+];
+
+export const Story = () => (
```

Comment:
- We don't want _Story_ in the name of the story - it's redundant. Let's call it `Typography` instead. We'll need to rename the actual component so that will work.
- We also don't want to needlessly nest, since there's only one story here we don't need `/Typography` in the title.